Framework/src/runAdvanced.cxx

```diff
@@ -0,0 +1,152 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    runAdvanced.cxx
+/// \author  Piotr Konopka
+///
+/// \brief This is an executable showing a more complicated QC topology.
+///
+/// This is an executable showing a more complicated QC topology. It pretends to spawn 4 separate topologies - 3 of them
+/// consist of some dummy processing chain, a dispatcher and a local QC task. The last one represents the remote
+/// QC servers topology, which has a merger (joining the results from local QC tasks), a checker (checks the result of
+/// the previous) and a different, remote QC task with associated checker. Here they are joined into one big topology
+/// just to present the concept.
+
+/// To launch it, build the project, load the environment and run the executable:
+///   \code{.sh}
+///   > aliBuild build QualityControl --defaults o2
+///   > alienv enter QualityControl/latest
+///   > qcRunAdvanced
+///   \endcode
+/// If you have glfw installed, you should see a window with the workflow visualization and sub-windows for each Data
+/// Processor where their logs can be seen. The processing will continue until the main window it is closed. Regardless
+/// of glfw being installed or not, in the terminal all the logs will be shown as well.
+
+#include <Framework/CompletionPolicyHelpers.h>
+#include "Framework/DataSampling.h"
+
+using namespace o2::framework;
+
+void customize(std::vector<CompletionPolicy>& policies)
```

Comment:
what is this ?

Reply:
This is some additional configuration of the topology, which is done by implementing `customize` functions and placing them **above**  `runDataProcessing.h` header. Personally, I am not a fan of this interface :)
In this case, both Dispatcher and Merger are configured to accept incoming messages without waiting for the rest of inputs.

Reply:
I see. Could you comment it ? no need to make it long.

Reply:
OK